tests/sync/test_settings.py

```diff
@@ -1,11 +1,38 @@
+import os
 import pathlib
 
 import pytest
 
 from kitovu import utils
+from kitovu.sync import settings
 from kitovu.sync.settings import Settings, ConnectionSettings
 
 
+@pytest.fixture
+def default_config(monkeypatch, temppath):
+    monkeypatch.setattr(settings.appdirs, 'user_config_dir', lambda _name: str(temppath))
+    return temppath / 'kitovu.yaml'
+
+
+def test_creating_default_config(default_config, monkeypatch):
+    monkeypatch.setattr(settings.subprocess, 'call', lambda args: None)
+    assert not default_config.exists()
+    spawner = settings.EditorSpawner()
+    spawner.edit()
+    assert default_config.exists()
+
+
+def test_load_unreadable_file(temppath):
+    settings = temppath / 'kitovu.yaml'
+    settings.touch()
+    settings.chmod(0)
+    if os.access(settings, os.R_OK):
+        pytest.skip("Failed to make file unreadable")  # pragma: no cover
```

Comment:
This can happen in Docker containers or weird filesystems, so we just skip the rest of the test there.